src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1334,6 +1334,10 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
     self->flags                            = 0;
     kh_init_inplace(uct_dc_mlx5_fc_hash, &self->tx.fc_hash);
 
+#if UCS_ENABLE_ASSERT
+    self->tx.allow_misordering             = 0;
```

Comment:
it can be dc_iface flag (in self->flags) then no need special debug-mode handling

Reply:
> it can be dc_iface flag (in self->flags) then no need special debug-mode handling

used flags